src/components/CortexLayout/CortexLayout.tsx

```diff
@@ -77,11 +77,27 @@ export const CortexLayout = ({
 }: CortexLayoutProps) => {
   const routes = createSubRoutesFromChildren(children);
 
+  const config = useApi(configApiRef);
+  const headerConfig = config.getOptionalConfig('cortex.header');
+  const customTitle = headerConfig?.getOptionalString('title');
+
   return (
     <Page themeId="home">
       <Header
-        title={title ?? 'Explore our ecosystem'}
-        subtitle={subtitle ?? 'Discover solutions available in our ecosystem'}
+        title={title
+          ?? (
+            customTitle
+            ? (
+            <>
+              {customTitle}
+              <Typography variant='subtitle1'>Powered by Cortex</Typography>
+            </>
+            )
+            : 'Cortex'
```

Comment:
Why fall back in the component instead of default values in the config? This seems like a leaky abstraction that's more difficult to understand and maintain.

Reply:
If custom title is set, we want to display "Powered by Cortex" under that. I refactored a bit so it's nicer.